pyannotate_tools/annotations/main.py

```diff
@@ -1,8 +1,9 @@
 """Main entry point to mypy annotation inference utility."""
 
 import json
+from io import IOBase
```

Comment:
I wouldn't do that. I'd import IO from typing and use IO[str] as the type of the streams everywhere.